### DISCUSSION TITLE
List of all possible environment variables

### DISCUSSION THREAD

**Discussion by vitormfgoncalves** [chaskiq/chaskiq#842] [General]
Hi.

I was wondering if there is a list of all possible environment variables anywere.
The list from the help page if far from complete. Sometimes , we have to search in the issues/discussions/code to find what we are loking for.
It would be very helpful!

**Comment by michelson** [comment]
is there any specific feature you are looking for?

**Reply by vitormfgoncalves** [reply]
Right now, I'm not looking for any speciffic feature.
I searched for hours and many frustrations later, found everything I was loking for.
I think that this list would be very helpfull for everyone trying to install this software.

As an axample, I use navidrome, and they have a list of all possible variables with descriptions that makes the install process much easier. Take a look: [https://www.navidrome.org/docs/usage/configuration-options/](https://www.navidrome.org/docs/usage/configuration-options/)

**Reply by michelson** [reply]
did you read this article? https://dev.chaskiq.io/production-configuration/enviroment-variables-configuration

**Reply by vitormfgoncalves** [reply]
Yes, I did. And it doesn't include a lot of usefull variables. Only the required ones.

A few examples:
SMTP_DELIVERY_METHOD
SMTP_ADDRESS
SMTP_USERNAME
SMTP_PASSWORD
ACTIVE_STORAGE_SERVICE
LOCAL_STORAGE_PATH
DISABLE_APP_CREATION
